src/content/app/genome-browser/state/track-settings/trackSettingsSlice.ts

```diff
@@ -221,6 +221,12 @@ const trackSettingsSlice = createSlice({
       }>
     ) {
       const { genomeId, trackSettings } = action.payload;
+
+      // Show track names by default for all tracks
+      for (const track of Object.values(trackSettings)) {
+        track.settings.name = true;
```

Comment:
Works as expected, except it overwrites the user preference after reload (compare to e.g. "show feature labels", another global track setting). @azangru where do default track settings live (`defaultTrackSettingsForGenome` is empty)?

Reply:
I guess this is the right place to configure default settings [useGenomeBrowserTracks.ts#L172](https://github.com/Ensembl/ensembl-client/blob/main/src/content/app/genome-browser/hooks/useGenomeBrowserTracks.ts#L172). Please confirm